src/app/components/chat/invite-modal/invite-modal.vue

```diff
@@ -16,6 +16,7 @@
 			</div>
 
 			<div class="modal-body">
+				<input class="-filter form-control" placeholder="Title" />
```

Comment:
This doesn't appear to be tied to anything. Trying to initialize a group chat with a room title doesn't name it.

Reply:
This component will probably need to be changed to an `app-form` input field.